### ISSUE TITLE
NTR: Nails: splinter hemorrhages (oriented base to tip) (Extremities, joints & nails)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7979]
New term request Nails: splinter hemorrhages (oriented base to tip) (brumme_220415200910): Typically embolic, haemorrhage  UMLS: 0474374

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000139
- label: Splinter hemorrhages
- parents: [Abnormality of the nail (HP:0001597)](https://hpo.jax.org/app/browse/term/HP:0001597)
- pmid: PMID:36050132
- def: Thin, red to reddish-brown lines of blood (hemorrhages) under the nails. Splinter hemorrhages run vertically under the nails.
- comment: Splinter hemorrhages represent rupture of longitudinally oriented nail plate capillaries leading to extravasation and the historical hallmark of linear discoloration. See figure 1 of PMID:36050132.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7979
- synonyms:
